advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,309 @@
+
+![ Refresh Available Tables](Images/refreshtables.png)
+
+22. To display the available tables click newly added `Server` —> `Tables` —> `Available Tables` —> `Public` (Schema name), a list of available tables will be displayed.
+
+![Available Tables](Images/availabletables.png)
+
+## Configure Postgres Distribution as Target Instance in Connect CDC Director
+
+The following steps demonstrate the setup and configuration of EDB Postgres Advanced Server as target instance to Connect CDC Director. 
+
+1. Right click `DBMS Servers` and `New Server`.
+
+2. Provide Connection Information for the EDB Postgres Advanced Server Target Instance.
+
+![Target Server](Images/targetserver.png)
+
+3. Right Click on the newly added `Server` and select `Install Target Only Metabase`.
+
+4. Provide EDB Postgres Advanced Server Target Instance User and Password on `User ID and Password` Screen.
+
+5. Provide password of `rpuser` (refer to the user name and password given on `New Server` Screen) when prompted on the `User ID and Password` screen.
+
+6. Click Ok on `Install Target Only Metabase` screen.
+
+7. A success message will be displayed once the `Target Only Metabase` is created.
+
+8. Open pgAdmin and connect to EDB Postgres Advanced Server `Target` Instance, and under Schemas `rpuser` Schema is created.
+
+![pgAdmin](Images/targetpgadmin.png)
+
+9. Right Click on the newly added `Server` and select `Prepare User Database`, this will add the replication user to EDB Postgres Advanced Server Target Instance.
+
+10. On `Database/Schema Name` provide schema name `public`.
+
+11. On `User ID and Password` Screen, provide EDB Postgres Advanced Server Target Instance user and password.
+
+12. Click Ok on the `Prepare User Database` screen.
+
+13. A success message will be displayed once the `Prepare User Database` is successful.
+
+## Create Distribution on Connect CDC Director
+
+The following steps demonstrate the creation of Distribution which will define how data will be replicated from source to target. In our case from EDB Postgres Advanced Server source to EDB Postgres Advanced Server target.
+
+1. Open Connect CDC Director.
+
+2. Right click on `Distributions` and select `New Distribution`.
+
+3. Enter Distribution information e.g `distribution name`, `description` etc and click ok.
+
+![New Distribution](Images/newdistribution.png)
+
+4. Go to Source DBMS `Server` —> `Tables` —> `Available Tables` —> `Public` (Schema name).
+
+5. Select the table(s) to replicate using the checking box in front of them and right click and select `Select for Distribution` —> `All Checked Tables`.
+
+![Select Tables for Replication](Images/selecttableforreplication.png)
+
+6. On the `Distributed Tables` Screen, select the `Target Server` under `Select one or more target servers` and select `Create tables on target` option and select the distribution under `Select a Distribution` option and click ok.
+
+![Distributed Tables](Images/distributetables.png)
```

Comment:
Crop some of the white space out of the bottom.

Reply:
done